doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -3075,9 +3079,104 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
 
     DELETE http://$SERVER/api/admin/authenticatedUsers/id/$id
     
-Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted.
-    
-    
+Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted. To see where in the database these actions are stored you can use the :ref:`show-user-traces-api` API. If a user cannot be deleted for this reason, you can choose to :ref:`disable a user<disable-a-user>`.
+
+.. _disable-a-user:
+
+Disable a User
+~~~~~~~~~~~~~~
+
+Disables a user.
+
+.. note:: See :ref:`curl-examples-and-environment-variables` if you are unfamiliar with the use of export below.
+
+.. code-block:: bash
+
+  export API_TOKEN=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+  export SERVER_URL=http://localhost:8080
+  export USERNAME=jdoe
+
+  curl -H "X-Dataverse-key:$API_TOKEN" -X POST $SERVER_URL/api/admin/authenticatedUsers/$USERNAME/disable
```

Comment:
Why would it be necessary to use a token here? IIRC all other endpoints in admin API are used without these tokens. For security, you can either use it on localhost only or add the `?unblock-key=...` feature. This might cause confusion.

Reply:
It's because DisableUserCommand calls RevokeAllRolesCommand, which requires superuser permissions.

Reply:
We should think of a strategy here to not require a token for an admin api, as I agree it's confusing. Generally (I think this is true), things in the admin API do not call commands, as commands require users. I think that has been a good model (or we need to decide on having commands that don't require users, but that feels dangerous).

Reply:
Fixed in d6e11732c (using authSvc.getAdminUser as suggested elsewhere).